sn_registers/src/register.rs

```diff
@@ -141,7 +139,7 @@ impl Register {
     /// Create a new Register
     pub fn new(owner: PublicKey, meta: XorName, mut permissions: Permissions) -> Self {
```

Comment:
we could replace the function name with an overview of what the comment above is saying, removing the need for the comment and making the function name more descriptive